ts/types/wallet.ts

```diff
@@ -14,14 +14,15 @@ export type WalletTransaction = {
   id: number;
   cardId: number;
   isoDatetime: string;
-  date: string;
-  time: string;
+  date: string; // it could be the same of date, having date as date.toLocaleDateString() TODO: @https://www.pivotaltracker.com/n/projects/2048617/stories/157769657
+  time: string; // it could be the same of date, having date as date.toLocaleTimeString() TODO: @https://www.pivotaltracker.com/n/projects/2048617/stories/157769657
   paymentReason: string;
   recipient: string;
   amount: number;
   currency: string;
-  transactionCost: number;
+  transactionCost: number; // it should be amount + transactionCost TODO: @https://www.pivotaltracker.com/n/projects/2048617/stories/157769657
   isNew: boolean;
+  isTransactionCompleted: boolean; // it will be true when the user accepted to proceed with a transaction and he is going to display the datail of the transaction as receipt
```

Comment:
This is probably not part of this interface. Rather, the `isTransactionCompleted` flag should be part of the state of the app